GatherPress/gatherpress
# Improve translations

### Description of the Change

This PR fixes four little bugs according to the translation handling of some strings.

1. >**Dateformat of datetime block is not translated/translatable**
I think @patriciabt mentioned this as well. Not sure if a ticket was created though.
(*from https://github.com/GatherPress/gatherpress/issues/557*)
2. #551 
3. #551 
4. #551 


Closes #551

### How to test the Change

Changes can be seen & tested, when the translations are updated, so every translation contributor is happily invited to:
1. Update their .po files from the updated .pot file
2. Update and add translations 
3. Run `npm run i18n:json`
4. Run `npm run i18n:php`

 (*This should be an issue, on its own ... i feel it*)

### Changelog Entry

> Fixed - Render translated version of date-time-block also within the editor and simplify translation of (some) strings 


### Credits

Props @carstingaxion 

### Checklist:
